src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Success');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Error'));
+  }, 3000);
+});
+
+promise1
+  .then((result) => {
+    const body = document.body;
+    const div = document.createElement('div');
+
+    div.classList.add('message');
+    div.textContent = 'Promise was resolved!';
+    body.appendChild(div);
```

Comment:
better to create some common function and move that logic there

and just call that function everywhere